scripts/validate_docstrings.py

```diff
@@ -679,5 +760,24 @@ def header(title, width=80, char='#'):
                            nargs='?',
                            default=None,
                            help=func_help)
+    argparser.add_argument('--format', default='default', help='format of the '
+                           'output when validating multiple docstrings '
+                           '(ignored when validating one). '
+                           'It can be {}'.format(str(format_opts)[1:-1]))
+    argparser.add_argument('--prefix', default=None, help='pattern for the '
+                           'docstring names, in order to decide which ones '
+                           'will be validated. A prefix "pandas.Series.str.'
+                           'will make the script validate all the docstrings'
+                           'of methods starting by this pattern. It is '
+                           'ignored if parameter function is provided')
+    argparser.add_argument('--errors', default=None, help='comma separated '
+                           'list of error codes to validate. By default it '
+                           'validates all errors (ignored when validating '
+                           'a single docstring)')
+
     args = argparser.parse_args()
-    sys.exit(main(args.function, sys.stdout))
+    if args.format not in format_opts:
+        raise ValueError('--format argument must be one of {}'.format(
+            str(format_opts)[1:-1]))
```

Comment:
You have this validation for `format` in two separate locations.  Why not just let the error propagate instead of doing the check here as well?